### DISCUSSION TITLE
Switching From Local Container To Remote

### DISCUSSION THREAD

**Discussion by guido331** [blakeblackshear/frigate#1207] [Ask A Question] [answered]
Hi,

I have been running my Frigate as an addon in hass.io using local CPU detection. Now that I have USB Corals I am running Frigate on a separate server for the direct hardware access. When I go to install and then start Frigate in hass.io it doesn't give me an option to enter the host for my Frigate instance. 

Where can I go to configure Frigate to connect to a remote host rather than attempting to run locally in hass.io?

**Comment by blakeblackshear** [comment]
> When I go to install and then start Frigate in hass.io it doesn't give me an option to enter the host for my Frigate instance

Are you referring to the custom component (integration)? You do not need to install the addon if you are running frigate elsewhere.

**Comment by guido331** [comment]
Yes - the integration, is there a way to get that working if Frigate is running on a separate workstation?

**Comment by blakeblackshear** [comment]
Yes. Many people are doing that. In home assistant, go to Configuration > Integrations > click the + > Select "Frigate". That should prompt for a url.

**Comment by guido331** [comment]
Apologies if this is a newb question, I don't have Frigate in my list of available integrations. (goes from Freebox to AVM FritzSmartHome) I have the github repository added in the add-on store and can install Frigate from there, but that installs everything and runs it locally. 

So far I'm loving the performance with the USB Corals, just need to get this integration part working and I'll be all set.

**Reply by highaltidude** [reply]
You'll also have to add Frigate in the customer repositories with HACS, then it should show as an available integration under Configuration>Integrations.

**Comment by blakeblackshear** [comment, marked as answer]
You are getting addons and Integrations confused. They are two different things. You need to install the integration into home assistant as described here: https://github.com/blakeblackshear/frigate-hass-integration

Addons run other software as containers alongside home assistant. Integrations "integrate" other software into home assistant natively.
